data/i18n/hi/hi.toml

```
@@ -1,180 +1,179 @@
 # i18n strings for the English (main) site.
 
 [deprecation_warning]
-other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the "
+other = " प्रलेखन अब सक्रिय रूप से मेंटेन नहीं है। वर्तमान में आप जो संस्करण देख रहे हैं वह एक स्टैटिक स्नैपशॉट है। अप-टू-डेट प्रलेखन के लिए, देखें "
 
 [deprecation_file_warning]
-other = "Deprecated"
+other = "डेप्रिकॉटेड"
 
 [objectives_heading]
-other = "Objectives"
+other = "उद्देश्यों"
 
 [cleanup_heading]
-other = "Cleaning up"
+other = "सफाई करना"
 
 [prerequisites_heading]
-other = "Before you begin"
+other = "शुरू करने से पहले"
 
 [whatsnext_heading]
-other = "What's next"
+other = "आगे क्या है"
 
 [feedback_heading]
-other = "Feedback"
+other = "प्रतिक्रिया"
 
 [feedback_question]
-other = "Was this page helpful?"
+other = "क्या यह पृष्ठ उपयोगी था?"
 
 [feedback_yes]
-other = "Yes"
+other = "हाँ"
 
 [feedback_no]
-other = "No"
+other = "नहीं"
 
 [latest_version]
-other = "latest version."
+other = "नवीनतम संस्करण।"
 
 [version_check_mustbe]
-other = "Your Kubernetes server must be version "
+other = "आपका कुबेरनेट्स सर्वर संस्करण होना चाहिए "
 
 [version_check_mustbeorlater]
-other = "Your Kubernetes server must be at or later than version "
+other = "आपका कुबेरनेट्स सर्वर पर या बाद में संस्करण की तुलना में होना चाहिए "
 
 [version_check_tocheck]
-other = "To check the version, enter "
+other = "संस्करण की जांच करने के लिए, दर्ज करें "
 
 [caution]
-other = "Caution:"
+other = "सावधानी:"
 
 [note]
-other = "Note:"
+other = "टिप्पणी:"
 
 [warning]
-other = "Warning:"
+other = "चेतावनी:"
 
 [main_read_about]
-other = "Read about"
+other = "के बारे में पढ़ा"
 
 [main_read_more]
-other = "Read more"
+other = "अधिक पढ़ें"
 
 [main_github_invite]
-other = "Interested in hacking on the core Kubernetes code base?"
+other = "कोर कुबेरनेट्स कोड बेस पर हैकिंग के इच्छुक हैं?"
 
 [main_github_view_on]
-other = "View On GitHub"
+other = "गिटहब पर देखें"
 
 [main_github_create_an_issue]
-other = "Create an Issue"
+other = "एक इशू बनाएं"
```

I think this is more sensible
```suggestion
other = "इशू बनाएं"
```

---

Yes! I think this suggestion looks good.